detox/src/devices/drivers/android/genycloud/helpers/GenyCloudInstanceAllocation.js

```diff
@@ -1,18 +1,17 @@
 const AndroidDeviceAllocation = require('../../AndroidDeviceAllocation');
-const GenyCloudInstanceHandle = require('../GenyCloudInstanceHandle');
 const DetoxRuntimeError = require('../../../../../errors/DetoxRuntimeError');
 const retry = require('../../../../../utils/retry');
 const logger = require('../../../../../utils/logger').child({ __filename });
 
 const { ALLOCATE_DEVICE_LOG_EVT } = AndroidDeviceAllocation;
 
 class GenyCloudInstanceAllocation extends AndroidDeviceAllocation {
-  constructor(deviceRegistry, deviceCleanupRegistry, instanceLookupService, instanceLifecycleService, eventEmitter) {
+  constructor(deviceRegistry, instanceLookupService, instanceLifecycleService, instanceLauncher, eventEmitter) {
```

Comment:
Too many args in the constructor to my taste, I'd recommend using destructuring instead.